src/message_filter.rs

```diff
@@ -6,15 +6,15 @@
 // KIND, either express or implied. Please review the Licences for the specific language governing
 // permissions and limitations relating to use of the SAFE Network Software.
 
+use crate::cache::Cache;
 use crate::messages::RoutingMsgUtils;
-use lru_time_cache::LruCache;
 use sn_messaging::{node::RoutingMsg, DstLocation, MessageId};
 use std::time::Duration;
 use xor_name::XorName;
 
 const INCOMING_EXPIRY_DURATION: Duration = Duration::from_secs(20 * 60);
 const OUTGOING_EXPIRY_DURATION: Duration = Duration::from_secs(10 * 60);
-const MAX_ENTRIES: usize = 15_000;
+//const MAX_ENTRIES: usize = 15_000;
```

Comment:
removed? or at least put a TODO comment

Reply:
Will be addressing all cache things (still wip) in follow up PR.